packages/gatsby-recipes/README.md

```diff
@@ -201,6 +201,16 @@ Installs a Gatsby Plugin in the site's `gatsby-config.js`.
 
 > Note that this content is stored in a [GitHub gist](https://gist.github.com/). When linking to a gist you'll want to click on the "Raw" button and copy the URL from that page.
 
+### `<Directory>`
+
+```jsx
+<Directory path="test" />
+```
+
+#### props
+
+- **path**: path to the directory that should be created. The path is local to the root of the Node.js project (where the package.json is)
```

Comment:
```suggestion
- **path**: path to the directory that should be created. The path is local to the root of the Node.js project (where the `package.json` is)
```

Reply:
Thanks, @muescha. There was another instance of `package.json` a few lines above this so I made both changes at once and pushed from my local.